server/grpc_service.go

```diff
@@ -1888,9 +1888,9 @@ func (s *GrpcServer) StoreGlobalConfig(_ context.Context, request *pdpb.StoreGlo
 		name := item.GetName()
 		switch item.GetKind() {
 		case pdpb.EventType_PUT:
-			ops[i] = clientv3.OpPut(s.GetFinalPathWithinPD(request.GetConfigPath()+name), item.GetValue())
+			ops[i] = clientv3.OpPut(request.GetConfigPath()+name, item.GetValue())
```

Comment:
nit: this line, compared to line 1893, has different coding style

Reply:
"...)+name vs. "...) + name"

Reply:
This style is generated by go fmt